app/controllers/locations_controller.rb

```diff
@@ -0,0 +1,21 @@
+class LocationsController < ApplicationController
+  def update
+    @location = find_location
+
+    @location.update!(location_params)
+
+    flash[:success] = t(".success")
+
+    redirect_to profile_url
```

Comment:
Can you write this as one line?

`redirect_to profile_url, success: t(".success")`